### DISCUSSION TITLE
How to use with nvchad.com?

### DISCUSSION THREAD

**Discussion by mainrs** [catppuccin/nvim#603] [Q&A]
Hello!

I am trying to enable the theme with [nvchad.com](https://nvchad.com) together. But for some reason neither the status bar nor nvim-tree are colorized with the `latte` theme.

This is my config and how I load the plugin:

```lua
{ 
    "catppuccin/nvim",
    name = "catppuccin",
    lazy = false,
    priority = 1000,
    config = function()
      vim.g.catppuccin_flavour = "latte"
      require("catppuccin").setup {
        flavour = "latte",
        dim_inactive = {
          enabled = false,
        },
        integrations = {
          cmp = true,
          nvimtree = true,
          treesitter = true,
        },
      }

      vim.cmd.colorscheme "catppuccin-latte"
    end
  },
```

This is how it looks like on my system:

![image](https://github.com/catppuccin/nvim/assets/5113257/2c5b702a-6072-4679-bb9e-f51c7527571b)

As you can also see, the terminal colors are the one of the non-latte theme; they are pastell. What am I doing wrong?

__Edit:__ I did some more digging into the nvchad.com source. [Here](https://github.com/NvChad/NvChad/blob/v2.0/lua/plugins/configs/nvimtree.lua) is the default configuration of nvim-tree that gets loaded [here](https://github.com/NvChad/NvChad/blob/a8413849cf9d648df0ae81a9349c8b125e628919/lua/plugins/init.lua#L216-L228). Not sure if this clashes somehow.

**Comment by FelipeEmos** [comment]
I'm facing the same problem...
Did you see how they setup their default themes?
They use this specific plugin made for `NvChad` that setus up the variables in a different way so that they can be used alongside the whole ecosystem ( telescope etc )

https://github.com/NvChad/base46/tree/v2.0

Here's their catppuccin Mocha implementation:
https://github.com/NvChad/base46/blob/v2.0/lua/base46/themes/catppuccin.lua

**Comment by vollowx** [comment]
NvChad doesn't seem to support themes other from the integrated ones by **base46**. As they integrated **catpouccin-mocha**, the best way is to just use it.

However, if you really want to use this plugin, you should firstly try to disable the plugin **NvChad/base46**, and manually define custom highlights for the statusline, tab/bufferline, **NvTerm**, etc for they don't exist. (Which might be a hard work!)

**Reply by FelipeEmos** [reply]
Thanks for the reply, you're 100% right.
It's very hard work to setup another flavor of catppuccin in NVChad because of those details.... I decided to just give up and move my setup to AstroNVim

Best decision ever 🚀
Their [Community Plugins](https://github.com/AstroNvim/AstroCommunity) feature made it incredibly easy to setup, basically the same effort as installing a couple of VSCode Extensions. They can be used to install things like Catppuccin Colorscheme or more complex stuff like a specific Language Support (LSPs, DAPs ... etc )

So shout out to anyone trying to setup a non standard NVChad colorscheme.... it's not worth it, you're basically fighting agains the NVChad distro 🙏🏻
